app/install/kubernetes.md

```diff
@@ -66,37 +74,49 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
     $ kubectl create -f postgres.yaml
     ```
 
-4. **Deploy Kong**
+4. **Prepare database**
+
+    Using the `kong_migration_<postgres|cassandra>.yaml` file from this repo,
+    run the migration job, jump to step 5 if Kong backing databse is up–to–date:
+    
+    ```bash
+    $ kubectl create -f kong_migration_<postgres|cassandra>.yaml
+    ```
+    Once job completes, you can remove the pod by running following command:
+
+    ```bash
+    $ kubectl delete -f kong_migration_<postgres|cassandra>.yaml
+    ```
+
+5. **Deploy Kong**
 
-    Using the `kong_<postgres|cassandra>.yaml` file from the
-    kong-dist-kubernetes repo, deploy a Kong `Service` and a `Deployment` to the
-    cluster created in the last step:
+    Using the `kong_<postgres|cassandra>.yaml` file from this
+    repo, deploy Kong admin and proxy services and a `Deployment` controller to
```

Comment:
"deploy Kong admin, proxy services, and a (...)"; what do we mean by "deploy Kong admin"? Could it be "deploy Kong"?